scripts/test-scripts/docker-linux-bind-mount-smoke.sh

```
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+# Linux-only Docker smoke test for host UID/GID bind-mount persistence.
+set -Eeuo pipefail
+
+readonly image="${MARM_DOCKER_SMOKE_IMAGE:-lyellr88/marm-mcp-server:latest}"
+readonly smoke_root="$(mktemp -d)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)